packages/usa-in-page-navigation/src/index.js

```diff
@@ -0,0 +1,219 @@
+
+const IN_PAGE_NAV_CLASS = `${PREFIX}-in-page-nav`;
+const IN_PAGE_NAV_LIST_CLASS = `${IN_PAGE_NAV_CLASS}-list`;
+const IN_PAGE_NAV_ITEM_CLASS = `${IN_PAGE_NAV_CLASS}__item`;
+const IN_PAGE_NAV_LINK_CLASS = `${IN_PAGE_NAV_CLASS}__link`;
+const IN_PAGE_NAV_ANCHOR_CLASS = "usa-anchor-tag";
+
+const IN_PAGE_NAV_LINK = `.${IN_PAGE_NAV_LINK_CLASS}`;
+const IN_PAGE_NAV_ANCHOR = `.${IN_PAGE_NAV_ANCHOR_CLASS}`;
+const IN_PAGE_NAV_TITLE = "On this page";
+const IN_PAGE_NAV = `.${PREFIX}-in-page-nav`;
+const CURRENT_CLASS = `${PREFIX}-current`;
+const MAIN_ELEMENT = "main";
+const SITE_SUBHEADING = "site-subheading";
+const SUB_ITEM = "sub-item";
+
+// Set Intersection Observer options
+const IO_ROOT = null;
+const IO_ROOT_MARGIN = "0px 0px -95% 0px";
+const IO_THRESHOLD = [0];
+
+/**
+ * Set the active link state for the currently observed section
+ *
+ * @param {HTMLElement} el An element within the in-page nav component
+ */
+const setActive = (el) => {
+  const allLinks = document.querySelectorAll(IN_PAGE_NAV_LINK);
+  el.map((i) => {
+    if (i.isIntersecting === true && i.intersectionRatio >= 1) {
+      allLinks.forEach((link) => link.classList.remove(CURRENT_CLASS));
+      document
+        .querySelector(`a[href="#${i.target.id}"]`)
+        .classList.add(CURRENT_CLASS);
+      return true;
+    }
+    return false;
+  });
+};
+
+/**
+ * Return a node list of section headings
+ *
+ * @return {HTMLElement[]} - An array of DOM nodes
+ */
+const getSectionHeadings = () => {
+  const sectionHeadings = document.querySelectorAll(
+    `${MAIN_ELEMENT} h2, ${MAIN_ELEMENT} h3`
+  );
+  return sectionHeadings;
+};
+
+/**
+ * Return a node list of section anchor tags
+ *
+ * @return {HTMLElement[]} - An array of DOM nodes
+ */
+const getSectionAnchors = () => {
+  const sectionAnchors = document.querySelectorAll(IN_PAGE_NAV_ANCHOR);
+  return sectionAnchors;
+};
+
+/**
+ * Return a section id/anchor hash without the number sign
+ *
+ * @return {String} - Id value with the number sign removed
+ */
+const getSectionId = (value) => {
+  let id;
+
+  // Check if value is an event or element and get the cleaned up id
+  if (value && typeof value === "object" && value.nodeType === 1) {
+    id = value.getAttribute("href").replace("#", "");
+  } else {
+    id = value.target.hash.replace("#", "");
+  }
+
+  return id;
+};
+
+/**
+ * Scroll smoothly to a section based on the passed in element
+ *
+ * @param {HTMLElement} - Id value with the number sign removed
+ */
+const handleScrollToSection = (el) => {
+  window.scroll({
+    behavior: "smooth",
+    top: el.offsetTop,
+    block: "start",
+  });
+};
+
+const createInPageNav = () => {
+  const sectionHeadings = getSectionHeadings();
+  const insertNode = document.querySelector(IN_PAGE_NAV);
+
+  const inPageNav = document.createElement("nav");
+  inPageNav.setAttribute("role", "navigation");
+  inPageNav.setAttribute("arial-label", "In-page navigation");
```

Comment:
Found a typo with aria-label. 
Additionally, the readout on the aria-label reads "In page navigation, navigation". We should be able to safely remove the word "navigation" from this attribute. (Reference: [Mozilla](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/navigation_role#associated_wai-aria_roles_states_and_properties))

```suggestion
  inPageNav.setAttribute("aria-label", IN_PAGE_NAV_TITLE );
```

Reply:
Changing to IN_PAGE_NAV_TITLE so the reader says "On this page" or whatever the title is changed to.